packages/components/src/components/lifecycleValidation.spec.ts

```diff
@@ -22,6 +23,17 @@ const tagNamesPublicWithProps = TAG_NAMES.filter(
 const tagNamesPublicWithoutProps = TAG_NAMES.filter(
   (tagName) => !getComponentMeta(tagName).isInternal && !getComponentMeta(tagName).props
 );
+const tagNamesWithPropsOfTypeObject = TAG_NAMES.filter((tagName) => {
+  return (
+    getComponentMeta(tagName).breakpointCustomizableProps ||
+    (getComponentMeta(tagName).allowedPropValues &&
+      Object.values(getComponentMeta(tagName).allowedPropValues)
+        .map((prop) => {
```

Comment:
Why so complicated and so much overhead with creating a new array? The same could be achieved with https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some?retiredLocale=de